src/xcode/ENA/ENA/Source/Scenes/StatisticsInfo/StatisticsInfoViewController.swift

```diff
@@ -237,6 +255,7 @@ class StatisticsInfoViewController: DynamicTableViewController {
 						text: AppStrings.Statistics.Info.trendTitle,
 						accessibilityIdentifier: nil
 					),
+					.space(height: 8),
```

Comment:
You change here the spacing for something other, is this intentional?

Reply:
Yes it is intentional, it's the space in trend text.